src/test/librbd/mock/MockImageCtx.h

```diff
@@ -237,6 +237,7 @@ struct MockImageCtx {
   ZTracer::Endpoint trace_endpoint;
 
   std::unique_ptr<crypto::MockEncryptionFormat> encryption_format;
+  uint64_t get_data_offset() const;
```

Comment:
Don't mix method and field declarations -- move it after `duplicate_data_io_context()` declaration.